server/routes/experiments/biomed_nl/entity_recognition.py

```diff
@@ -73,8 +81,8 @@ def sample_dcids_by_type(dcids, min_sample_size):
   num_skipped_samples_to_add = min_sample_size - len(sample_dcids)
   if num_skipped_samples_to_add > 0:
     sample_dcids.extend(first_pass_skipped[:num_skipped_samples_to_add])
-
-  return sample_dcids, unique_types
```

Comment:
do we still need to keep sample_dcids now that we have graph_entities?

Reply:
thanks for pointing this out, forgot to delete this after I confirmed the entity approach worked!